modules/olm-installing-operator-using-commands.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-adding-operators-to-cluster.adoc
+
+[id='olm-installing-operator-using-commands_{context}']
+
+= Installing an Operator using the command line
+
+Instead of using the {product-title} Web console, you can install an
+Operator from the command line. Use the `oc` command as follows to install
+(subscribe to) an Operator for a selected namespace.
+
+.Prerequisites
+
+- Access to an {product-title} 4 cluster using an account with `cluster-admin`
+permissions.
+
+.Procedure
+
+. To view a list of operators available from OperatorHub, type the following:
++
+----
+$ oc get packagemanifest
+----
+
+. To identify the Operators to install, create a CatalogSourceConfig object yaml file (for example, `csc.cr.yaml`)
+containing information similar to the following:
+
++
+.Example CatalogSourceConfig
+[source,yaml]
+----
+apiVersion: "marketplace.redhat.com/v1alpha1"
+kind: "CatalogSourceConfig"
+metadata:
+  name: "example"
+  namespace: "openshift-marketplace"
+spec:
+  targetNamespace: "global" <1>
+  packages: "operators/myoperator,operators/another-operator" <2>
+----
+<1> Set the `targetNamespace` to identify the namespace where you want the
+Operator to be available
+<2> Set `packages` to a comma-separated list of Operators to which
+you want to subscribe.
+
+. To install the Operator to the selected namespace, type the following:
++
+----
+$ oc apply -f csc.cr.yaml
+----
+
+. Later, if you want to install more operators, you can simply update your
+CatalogSourceConfig file (in this example, csc.cr.yaml) with more packages.
+For example:
++
+.Example updated CatalogSourceConfig
+[source,yaml]
+----
+apiVersion: "marketplace.redhat.com/v1alpha1"
+kind: "CatalogSourceConfig"
+metadata:
+  name: "example"
+  namespace: "openshift-marketplace"
+spec:
+  targetNamespace: "global"
+  packages: "operators/myoperator,operators/another-operator,operators/this-operator-too"
+ <1>
+----
+<1> Add new packages to existing package list.
+
+. To update the CatalogSourceConfig object, run the `oc apply` command again:
++
+----
+$ oc apply -f csc.cr.yaml
+----
++
+The Operator will generate a CatalogSource and ConfigMap from your
```

Comment:
We no longer generate a `ConfigMap` since moving to grpc `CatalogSources`. I suggest leaving that out.